deps/llvm.mk

```diff
@@ -336,6 +336,7 @@ endif
 ifneq ($(LLVM_LLDB_TAR),)
 	$(JLCHECKSUM) $(LLVM_LLDB_TAR)
 endif
+	-rm -rf $(LLVM_SRC_DIR)
```

Comment:
for the people who did rebuild from scratch recently, or haven't rebuilt since my commits, would touching cmakelists be equivalent?